Fix default value for grayscale() being 1, not 0 (#27940)
mdn/content: files/en-us/web/css/filter-function/grayscale/index.md

### Parameters

- `amount`

>   - : Amount of the input image that is converted to grayscale. It is specified as a {{cssxref("&lt;number&gt;")}} or a {{cssxref("&lt;percentage&gt;")}}. A value of `100%` changes the input completely to grayscale, while a value of `0%` leaves the input unchanged. Values between `0%` and `100%` have linear multipliers on the effect. The default value when omitted is `1`. The initial value used for {{Glossary("interpolation")}} is `0`.

Reviewer: ```suggestion
  - : Amount of the input image that is converted to grayscale. It is specified as a {{cssxref("&lt;number&gt;")}} or a {{cssxref("&lt;percentage&gt;")}}. A value of `100%` changes the input completely to grayscale, while a value of `0%` leaves the input unchanged. Values between `0%` and `100%` have linear multipliers on the effect. If the `grayscale()` filter is present with no parameter, the default value `1`. The initial value used for {{Glossary("interpolation")}} is `0`.
```
